editors-definitions/che-code-sshd-insiders.yaml

```diff
@@ -76,18 +76,26 @@ metadata:
       </defs>
       </svg>
 
+commands:
+  - id: start-sshd
+    exec:
+      component: che-code-sshd
+      commandLine: nohup /sshd/sshd.start &
+events:
+  postStart:
+    - start-sshd
 components:
-  - name: che-code-sshd
+  - name: che-code-sshd-page
     container:
       image: quay.io/che-incubator/che-code-sshd:insiders
-      memoryLimit: 4096Mi
-      memoryRequest: 256Mi
-      cpuLimit: 1000m
-      cpuRequest: 500m
+      memoryLimit: 256Mi
+      memoryRequest: 32Mi
+      cpuLimit: 500m
+      cpuRequest: 30m
       command:
         - sh
         - -c
-        - "nohup /entrypoint.sh & nohup /sshd.start & pushd /opt/www/ && nohup node /opt/www/server.js & tail -f /dev/null"
+        - "cp /usr/sbin/sshd /usr/bin/ssh-keygen /usr/lib64/libnss_wrapper.so /etc/ssh/sshd_config /sshd.start /sshd/ & while [ ! -e /sshd/username ]; do sleep 1s; done; pushd /opt/www/ && nohup node /opt/www/server.js & tail -f /dev/null"
```

Comment:
The `nohup node /opt/ww/server.js & tail -f /dev/null` needs some inspection. It could be simplified. Maybe prepending `exec` will keep things up without the need for the other part.